Add `npm install` instruction at the _Running the CLI_ section (#26)
github/ghec-audit-log-cli: README.md

## Running the CLI

> Execute the command using node or npm. Before to execute, run `npm install` to install dependencies.
> 
> ```shell script
> $ git clone https://github.com/github/ghec-audit-log-cli
> $ cd ghec-audit-log-cli
> $ npm install
> ```

Reviewer: ```suggestion
Execute the command using node or npm

### Pre-requisites

 Install the node dependencies:

```shell script
$ git clone https://github.com/github/ghec-audit-log-cli
$ cd ghec-audit-log-cli
$ npm install
```
```
